pkg/apiserver/controllers/v1/alerts.go

```diff
@@ -135,9 +135,9 @@ func (c *Controller) CreateAlert(gctx *gin.Context) {
 	}
 	select {
 	case c.CAPIChan <- input:
-		log.Debugf("alert send to CAPI channel")
+		log.Debug("alert send to CAPI channel")
 	default:
-		log.Warningf("Cannot send alert to Central API channel")
+		log.Debug("Cannot send alert to Central API channel")
```

Comment:
Hello!

Sorry for the late review, can you let this line in `Warningf` please?

Reply:
That was the main reason for me forking Crowdsec: every single decision on my server triggers this log.
IMO the warning at start/reload saying the CAPI is disabled should be enough. Why do you want this to be a warning specifically?

Reply:
If you disable CAPI, indeed the local API shouldn't try to send the alert to the CAPI channel. I've opened an [issue](https://github.com/crowdsecurity/crowdsec/issues/907) and this will be fix in the next release.

When this will be fixed, if you see this log again while having CAPI enabled, this mean that there is a problem somewhere.

Reply:
Can you point me in the correct direction so I do the change?

Reply:
I force pushed a change that makes the log a warning again, but only tries to send the event if the channel isn't `nil`. Please tell me if you think the check should be done differently.

Reply:
Yes i think it should be good, thanks!